README.md

```diff
@@ -1,64 +1,53 @@
 # @ngrx/store
+RxJS powered state management inspired by Redux for Angular applications
```

Comment:
Maybe flip this wording to _RxJS powered state management for Angular applications, inspired by Redux._